smac/facade/smac_ac_facade.py

```diff
@@ -544,7 +542,8 @@ def __init__(self,
             'acquisition_func': acquisition_function_instance,
             'rng': rng,
             'restore_incumbent': restore_incumbent,
-            'random_configuration_chooser': random_configuration_chooser_instance
+            'random_configuration_chooser': random_configuration_chooser_instance,
+            'tae_runner': tae_runner_instance,
```

Comment:
The tae runner is currently an object, whose run method is what executes the config.

We need to somehow specify to the execute_ta_run_wrapper/future worker, what to execute, and the mechanism used right now is providing this object.

This object reaches smbo as an argument. To remove this from here, we have to then deprecate the ta runners in favor of the user providing a callable to execute the runs? It will still have to reach the smbo.

Reply:
My bad, I read wrongly and didn't see in the side-by-side diff that this was a new dict. Sorry for bothering here.